test/distributed/test_c10d_nccl.py

```diff
@@ -299,7 +305,7 @@ def test_abort_pg(self):
         def abortpg():
             c10d.distributed_c10d._get_default_group()._get_backend(
                 torch.device(device)
-            )._shutdown()
+            ).abort()
```

Comment:
technically should be a separate PR, is this a trivial change or what are the behavior differences?